emails/templates/emails/wrapped_email.html

```diff
@@ -136,41 +136,41 @@
               {% ftlmsg 'relay-email-byline-html' url=SITE_ORIGIN|add:'/accounts/profile/' attrs='class="container-link" style="margin-right: 30px;color: #FFFFFF;"' %}
             {% endif %}
           </span>
-          </p>  
-            
-        </td> 
-        <td class="header-block-right" width="50%" align="right"> 
+          </p>
+
+        </td>
+        <td class="header-block-right" width="50%" align="right">
           <p class="relay-trackers-removed" style="margin: 0 30px 0 0; display: inline-block; color: #FFFFFF;">
-            <img class="email-trackers-removed-icon" width="15" src="{{ SITE_ORIGIN }}/static/images/email-images/email-trackers-removed-icon.png" alt="email trackers removed icon"/>   
+            <img class="email-trackers-removed-icon" width="15" src="{{ SITE_ORIGIN }}/static/images/email-images/email-trackers-removed-icon.png" alt="email trackers removed icon"/>
             {% comment %}
-              Create this as a link if we have a report link to show 
+              Create this as a link if we have a report link to show
             {% endcomment %}
             {% if num_level_one_email_trackers_removed > 0 and tracker_report_link %}
-            <a class="container-link" href="{{ tracker_report_link  }}" style="color: #FFFFFF;"> 
+            <a class="container-link" href="{{ tracker_report_link  }}" style="color: #FFFFFF;">
               {% ftlmsg 'relay-email-trackers-removed' number=num_level_one_email_trackers_removed %}
             </a>
             {% else %}
               {% ftlmsg 'relay-email-trackers-removed' number=num_level_one_email_trackers_removed %}
             {% endif %}
-          </p> 
+          </p>
 
           {% with display_email|striptags|urlencode as mask_url %}
-          <p class="relay-mask" style="margin: 0; display: inline-block;"> 
-              {% if has_premium %}
-              <a class="container-link" href="{{ SITE_ORIGIN }}/accounts/profile/#{{ mask_url }}" style="color: #FFFFFF;"> 
+          <p class="relay-mask" style="margin: 0; display: inline-block;">
+              {% if has_premium or not in_premium_country %}
```

Comment:
suggestion (blocking): Remove this change. We redirect these users from `/premium` to `/premium/waitlist` and we use that wait-list to help inform us which countries we should expand into.

Reply:
This was the only usage of `in_premium_country`, so I was able to remove that as well. I left a comment in the template about why it is OK to send all users to `/premium`.